libs/playground/src/app/docs/get-started/page.mdx

```diff
@@ -33,6 +33,14 @@ npm install @trikinco/fullstack-components
 
 ### Step 1: Add the API handler route
 
+Note: if using NextJS 13 you must turn on server actions in your `next.config.js` file. You must remove this if you later upgrade to NextJS 14 or higher.
```

Comment:
maybe step 0?

Reply:
Or maybe it could just be added as a note or sub-bullet under "Using the Next.js Pages Router: pages/api/fsutils/[...fscomponents].ts"? I'm fine with either and I'm also happy to do a full round of rewriting and clarifying the docs as we go. 

FYI I'm adding some more details here as well https://github.com/trikinco/fullstack-components/pull/66/files